internal/parser/grammar.md

```diff
@@ -105,7 +105,7 @@ For the parser, the above grammar undergoes a process to have the left-recursion
     CallExpression = lparen ParameterList rparen .
     IndexExpression = lbrack Expression rbrack .
     UnaryExpression = PrimaryExpression
-                    | PrefixOperator PrimaryExpression .
+                    | PrefixOperator PostfixExpression .
```

Comment:
I think one other part has to change. It looks like the current EBNF gives higher precedence to unary operations than the postfix and it should be of lower precedence. I think `PipeExpression` has to be changed to `PipeExpression = UnaryExpression { PipeExpressionSuffix }`.